### DISCUSSION TITLE
I have a big announcement

### DISCUSSION THREAD

**Discussion by mgilangjanuar** [mgilangjanuar/teledrive#29] [Announcements]
First of all, I'm apologize for the clickbait in the title.

Then, I was frustrated with the web framework we used 😵 Honestly, I don't usually use Chakra UI as a component library. I think they're too hard to create a single layout.

So, I decided to create another branch in an experiment with [ant.design](https://ant.design/). Hopefully, I can finish the beta app for the waiting list users tomorrow 🚀 

Once again, I really thankful for @grgp and @Aqshola for making the super awesome page that I've ever seen. Can you guys give me your address to my email mgilangjanuar@gmail.com? I'd like to send you a little gift as my appreciation for your wonderful work!

**Comment by grgp** [comment]
Sounds good! Yeah, sorry if Chakra UI wasn't the right choice for this project, it can be unpredictable sometimes. ant.design should be good, though. I think you haven't pushed the branch yet, right?

@mgilangjanuar I'll try to do some work done on Sunday afternoon hehe sorry I haven't done anything since two weeks ago. If there's anything in particular you need help on please list them. 🙇

**Reply by mgilangjanuar** [reply]
Yeah sure, I will push that branch as soon as possible 🚀 And sure I really need your help 😆

**Comment by Aqshola** [comment]
Yeah no problem, sometimes we must experiment in something to find right things 🤣, maybe this weekend i will have some freetime so i will catch up 👍

**Reply by mgilangjanuar** [reply]
Thank you so much! 🍻
